masq_lib/src/shared_schema.rs

```diff
@@ -48,6 +48,18 @@ pub const EARNING_WALLET_HELP: &str =
      (case-insensitive). If you already have a derivation-path earning wallet, don't supply this. \
      If you have supplied an earning wallet address before, either don't supply it again or be \
      careful to supply exactly the same one you supplied before.";
+pub const EXIT_LOCATION_HELP: &str =
+    "Choose your Exit Location for access the internet. You can choose from all countries, available in \
+    your Neighborhood. \n\
```

Comment:
Establish a set of countries that your Node should try to use for exit Nodes. You should choose from the countries that host the Nodes in your Neighborhood. List the countries in order of preference, separated by vertical pipes (|). If your level of preference for a group of countries is the same, separate those countries by commas (,).